ports/megamanx816-bit/README.md

```diff
@@ -0,0 +1,14 @@
+## Notes
+
+https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ grab the zip from here, extract it and rename the File to "Megaman X8 FC.pck" after that put it in megamanx816-bit/gamedata folder
```

Comment:
Notes section is for thank you notes with links to the developer etc. 
Instructions go into port.json